services/wallet/bridge/hop.go

```diff
@@ -372,16 +346,230 @@ func (h *HopBridge) CalculateFees(from, to *params.Network, token *token.Token,
 		return nil, nil, err
 	}
 
+	h.bonderFee = &BonderFee{}
 	err = json.Unmarshal(response, h.bonderFee)
 	if err != nil {
 		return nil, nil, err
 	}
 
-	tokenFee := new(big.Int).Sub(h.bonderFee.AmountIn, h.bonderFee.EstimatedRecieved)
+	// Remove token fee from bonder fee as said here:
+	// https://docs.hop.exchange/v/developer-docs/api/api#get-v1-quote
+	// `bonderFee` - The suggested bonder fee for the amount in. The bonder fee also includes the cost of the destination transaction fee.
+	tokenFee := big.NewInt(0) //new(big.Int).Sub(h.bonderFee.AmountIn.Int, h.bonderFee.EstimatedRecieved.Int)
```

Comment:
if u always set tokenfee at 0 is it still useful?
I think celer don't use that and hope was the only one. (To verify)

Reply:
As I added the comment there, the bonder fee should include the token fee (from the Hop docs), and yes, I agree that we can remove it, but let's do that later. I left it there just cause I was not fully sure if we will need it later or not.